pkg/config/config_params.go

```diff
@@ -30,10 +30,11 @@ import (
 )
 
 var (
-	IfaceListRegexp = regexp.MustCompile(`^[a-zA-Z0-9_-]{1,15}(,[a-zA-Z0-9_-]{1,15})*$`)
-	AuthorityRegexp = regexp.MustCompile(`^[^:/]+:\d+$`)
-	HostnameRegexp  = regexp.MustCompile(`^[a-zA-Z0-9_.-]+$`)
-	StringRegexp    = regexp.MustCompile(`^.*$`)
+	IfaceListRegexp      = regexp.MustCompile(`^[a-zA-Z0-9_-]{1,15}(,[a-zA-Z0-9_-]{1,15})*$`)
+	AuthorityRegexp      = regexp.MustCompile(`^[^:/]+:\d+$`)
+	HostnameRegexp       = regexp.MustCompile(`^[a-zA-Z0-9_.-]+$`)
+	StringRegexp         = regexp.MustCompile(`^.*$`)
+	prometheusHostRegexp = regexp.MustCompile(`^[a-zA-Z0-9:._+-]{1,64}$`)
```

Comment:
For consistency, could you make this exported?

Reply:
Ok, I have done so.